src/maga/util/Console.java

```diff
@@ -1,5 +1,6 @@
 package maga.util;
 
+// should probably be deleted
```

Comment:
Må gerne slettes.

Reply:
fixed in: 5bfd32fc4f668a057efc1cbd6dcd8cc34d3e5f0a